src/tests.c

```diff
@@ -3384,6 +3384,72 @@ void run_group_decompress(void) {
 
 /***** ECMULT TESTS *****/
 
+void test_pre_g_table(const secp256k1_ge_storage * pre_g, size_t n) {
+    secp256k1_gej g2;
+    secp256k1_ge p, q, gg;
+    secp256k1_fe dpx, dpy, dqx, dqy;
+    size_t i;
+
+    CHECK(0 < n);
+    secp256k1_ge_from_storage(&p, &pre_g[0]);
+    secp256k1_gej_set_ge(&g2, &p);
+    secp256k1_gej_double_var(&g2, &g2, NULL);
+    secp256k1_ge_set_gej_var(&gg, &g2);
+    secp256k1_fe_verify(&p.x);
```

Comment:
Move these there verification lines up to immediately after the `ge_from_storage`.

Reply:
fixed.